perlcriticrc

```diff
@@ -1,6 +1,6 @@
 severity = 3
 verbose = 11
-theme = core + pbp + bugs + maintenance + cosmetic + complexity + security + tests + moose
+theme = (core && (pbp || bugs || maintenance || cosmetic || complexity || security || tests)) || moose
```

Comment:
I wonder if this change really belongs in the bundle rather than here.

Reply:
Yes, probably, but the bundle doesn't update this file once it exists, so you'd still need to update this here.

Reply:
FWIW, I've updated this in my bundle. You guys might want to follow along to changes to said bundle since the `MAXMIND` one is basically a fork of `DROLSKY`.